scripts/integrations/angular/src/app/binding/editor-binding.component.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license Copyright (c) 2003-2019, CKSource - Frederico Knabben. All rights reserved.
```

Comment:
The years here should be `2003-2020`. This also needs to be fixed in several other angular and vue files.